### PR TITLE
Suppress signature OK Logging #404

### PR DESCRIPTION
**Pull request by meridsa**
Added an option to suppress ok signature

### CODE REVIEW

**Comment by meridsa** [comment]
Linking #404

File: src/main/java/org/simplify4u/plugins/CheckMojo.java [hunk 1]
```diff
@@ -99,6 +99,12 @@ public static class VerificationResult {
     @Parameter(property = "pgpgverify.failWeakSignature", defaultValue = "false")
     private boolean failWeakSignature;
 
+    /**
+     * Suppress Signature OK
+     */
```
Comment (slawekjaranowski) [inline src/main/java/org/simplify4u/plugins/CheckMojo.java:104]: Please add `@since 1.17.0`

Comment (meridsa) [reply src/main/java/org/simplify4u/plugins/CheckMojo.java]: Done :)

**Review by slawekjaranowski — APPROVED** [review]

**Comment by meridsa** [comment]
Excellent, thanks!

**Pull request merged**